server/src/concepts/offer/useCases/createNewOffer.js

```diff
@@ -0,0 +1,34 @@
+import { createOffer } from "../repositories/commands.js";
+import { findCategoryByName } from "../../category/repositories/queries.js";
+import { findHeightByValue } from "../../height/repositories/queries.js";
+
+export async function createNewOffer(body) {
+  try {
+    const offer = await createOffer({
+      title: body.title,
+      description: body.description,
+      city: body.city,
+      location: body.location || {},
```

Comment:
I see now, that we do not have location in validationOffer, so it should be added there. Because when there is no field "location" in joi schema validation, this field cannot be in request